### DISCUSSION TITLE
Is it possible to hide for a filter in association list view in a record show page?

### DISCUSSION THREAD

**Discussion by hassyyy** [avo-hq/avo#886] [Help] [answered]
For example, let's say I have 2 resources - Credit Cards & Transactions
Both are associated to each other: Credit Cards has many transactions or Transaction belong to credit card

In transactions list/index page, I have added a filter field with all the credit cards as options and the first credit card as the default value. Hence, only the transactions belonging to the selected credit card will be displayed in the list view. Here is the query:
```sql
SELECT "transactions".* FROM "transactions" WHERE "transactions"."credit_card_id" = ? [["credit_card_id", 1]]
```

However, this is causing problem in the credit card detail/show page
When I view the detail/show page of the credit card which is not the default value in the filter, none of the transactions are displayed. Here is the query:
```sql
SELECT "transactions".* FROM "transactions" WHERE "transactions"."credit_card_id" = ? AND "transactions"."credit_card_id" = ?[["credit_card_id", 1], ["credit_card_id", 2]]
```

Is there a way to skip using a particular filter for associations list in the detail page? Or at least a way to reset the default value based on the page

**Comment by hassyyy** [comment, marked as answer]
Hey @adrianthedev 
Kindly help with this 🙏

**Reply by adrianthedev** [reply]
Hey. Apologies. This notification must have slipped through the cracks. 

Can you try and see if you could use the `params` to figure out if you're on the show or index page?

**Reply by hassyyy** [reply]
The value for action is same in both show. & index page for the associated resource(since it is an index of the records)
However, I made use of other params(like "id") and resolved the issue. Thanks a lot 🥂
